guides/introduction/quick_start_guide.md

```diff
@@ -202,86 +266,111 @@ $ curl \
 
 ```js
 const movies = require("./movies.json");
-meili.Index("movies").addDocuments(movies);
+index.addDocuments(movies).then((res) => console.log(res));
 ```
 
+[About this package](https://github.com/meilisearch/meilisearch-js/)
 :::
 
 ::: tab Ruby
 
 ```ruby
+require 'json'
+
+movies_json = File.read('movies.json')
+movies = JSON.parse(movies_json)
 index.add_documents(movies)
 ```
 
+[About this package](https://github.com/meilisearch/meilisearch-ruby/)
 :::
 
 ::: tab PHP
 
 ```php
-$index->addOrReplaceDocuments($movies);
+$movies_json = file_get_contents('movies.json');
+$movies = json_decode($movies_json);
+$index->addDocuments($movies);
 ```
 
+[About this package](https://github.com/meilisearch/meilisearch-php/)
 :::
 
 ::: tab Python
 
 ```python
-index = self.client.get_index(uid="movies")
+import json
+
 json_file = open('movies.json')
-data = json.load(json_file)
-response = index.add_documents(data)
+movies = json.load(json_file)
+index.add_documents(movies)
 ```
 
+[About this package](https://github.com/meilisearch/meilisearch-python/)
+:::
+
+::: tab Go
+
+```go
+import (
+    "encoding/json"
+    "io/ioutil"
+)
+```
+
+```go
+moviesJSON, _ := os.Open("movies.json")
+defer moviesJSON.Close()
+
+byteValue, _ := ioutil.ReadAll(moviesJSON)
+var movies []map[string]interface{}
+json.Unmarshal(byteValue, &movies)
+
+updateRes, _ := client.Documents("movies").AddOrUpdate(movies)
+fmt.Println(updateRes.UpdateID)
+```
+
+[About this package](https://github.com/meilisearch/meilisearch-go/)
 :::
 
 ::::
 
 ### Checking updates
 
-In MeiliSearch, most actions are asynchronous. This lets you stack actions. They will be executed in the order in which they were made.
+The document addition returns a simple JSON with only an `updateId`.
+
+This kind of **successful response** indicates that the operation has been taken into account, but it may not have been executed yet.
 
-You can [track the state of each action](/guides/advanced_guides/asynchronous_updates.md).
+You can check the status of the operation via the `updateId` and the [get update status route](/references/updates.md).
 
-## Searches
+Checking the update status is not a mandatory step to search through your documents but could be really useful in case of error.
```

Comment:
```suggestion
Checking the update status is not a mandatory step to search through your documents but could be really useful to understand what is happening in case of errors or unexpected behaviors.
```